server/src/main/java/org/apache/iotdb/db/engine/storagegroup/StorageGroupProcessor.java

```diff
@@ -457,7 +460,7 @@ private VersionController getVersionControllerByTimePartitionId(long timePartiti
     List<File> tsFiles = new ArrayList<>();
     List<File> upgradeFiles = new ArrayList<>();
     for (String baseDir : folders) {
-      File fileFolder = fsFactory.getFile(baseDir, storageGroupName);
+      File fileFolder = fsFactory.getFile(baseDir + File.separator + logicalStorageGroupName, storageGroupName);
```

Comment:
This conflicts with what you have written in `SlotTsFileFilter.java`, please confirm it.

Reply:
Just a naming problem, fixed~